packages/server/package.json

```diff
@@ -34,6 +34,7 @@
         "dotenv": "16.0.0",
         "fs-extra": "10.0.0",
         "geolite2-redist": "2.0.4",
+        "isomorphic-dompurify": "0.19.0",
```

Comment:
That dependency looks clean, but - because it's a runtime dependency that has a direct, significant impact on security - I'd rather not use an npm package for this unless it comes from an **absolutely reputable source**. Since it's only a couple lines of code, maybe we can just add those to our repo directly? The package is licensed as MIT, so that shouldn't be a problem.

Reply:
To be clear, I'm ok with using `dompurify`, but I'd rather have another vendor in between, so let's just add that wrapper code to our repo directly.